wurst/objects/items/MixingPot/Cure.wurst

```diff
@@ -0,0 +1,38 @@
+package Cure
+
+import PotionsDefinition
+import ID
+import ChannelAbilityPreset
+import ClosureEvents
+import InstantDummyCaster
+import Orders
+import ObjEditingUtils
+
+constant ABIL  = compiletime(ABIL_ID_GEN.next())
+
+//constant ABIL  = "A05K"
+constant ICON  = ICON_PATH.format("LesserClarityPotion")
+constant MODEL = "Models\\PotionGreenSmall.mdl"
+
+constant CAST_RANGE = 500.
+constant TARGET_ALLOWED = "friend,organic,self"
+constant TT = "This potion is said to be able to cure anything. Made from Athelas Seed."
+
+@compiletime function createAbil()
+    new ChannelAbilityPreset(ABIL, 1, true)
+    ..setName("Cure Potion")
+    ..setCheckDependencies(false)
+    ..setHeroAbility(false)
+    ..setItemAbility(true)
+    ..presetTargetTypes(Targettype.UNIT)
+    ..setManaCost(1, 0)
+    ..setCooldown(1, 0)
+    ..setCastRange(1, CAST_RANGE)
+    ..setEditorSuffix("(Wurst)")
```

Comment:
Don't do this. Wurst does it automatically and doing it manually is just clutter.

Reply:
Wurst does it automatically? I added this suffix to make sure the spell I checked in the GUI editor was made in wurst, I believed it would be clearer, well at least for me

Reply:
Wurst sets its a custom field that isn't visible from the object editor, but is included in the object editing output. Are you opening the built map in the World Editor to debug?

Reply:
Yes, in some case I used the World Editor to debug, I took this habit when I started recoding the troll spells, I didn't remove the GUI spell at first, so I added the suffix to make a diff between old and new spell in the GUI.